### DISCUSSION TITLE
Markdoc with shadcn/ui

### DISCUSSION THREAD

**Discussion by noxify** [markdoc/markdoc#408] [Show and tell]
Since we're using only tailwind in our team ( and their Syntax template with markdoc for our docs page ), 
I decided to create a template which uses shadcn/ui and markdoc.

Most of the work is done by others, I just created the wrapper components to make them available via the markdoc syntax.

You can find the template here: https://github.com/noxify/next-shadcn-markdoc

The template is based on a NextJS App Directory starter, uses lighter as syntax highlighter and shadcn/ui ( suprise suprise ;) )

What do you think?

**Comment by mfix-stripe** [comment]
This looks awesome @noxify 🙌 Nice work.

**Reply by noxify** [reply]
Thanks ❤️

Any idea what could be added as custom tag? Could spend some time next week to extend the template 😀 

Also would it make sense to make a dedicated npm package for the custom tags (one for all vs. one package per custom tag)?
